lib/proxy/verify_runtime.rb

```diff
@@ -0,0 +1,52 @@
+module Proxy
+  class VerifyRuntime
+    class << self
+      # RFC5737 declares 192.0.2.0/24 as TEST-NET-1
+      MOCK_IP = '192.0.2.42'
+
+      def settings
+        Proxy::SETTINGS
+      end
+
+      def verify
+        {
+          reverse_proxy: verify_reverse_proxy,
+        }
+      end
```

Comment:
I'm thinking about the format. Perhaps it should have some global status and then a hash for modules.
```json
{
  "global": {},
  "modules": {
    "templates": {
      "status": true,
      "reverse_proxy": true,
    },
    "registration": {
      "status": true,
    }
  }
}
```

Perhaps also a summary?

What if it doesn't? Should we use error codes? Human readable strings? both?